packages/mesa/build.sh

```diff
@@ -18,19 +19,30 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 -Dgbm=enabled
 -Dopengl=true
 -Degl=enabled
--Degl-native-platform=x11
 -Dgles1=disabled
 -Dgles2=enabled
 -Ddri3=enabled
 -Dglx=dri
 -Dllvm=enabled
 -Dshared-llvm=disabled
--Dplatforms=x11
--Dgallium-drivers=swrast
--Dvulkan-drivers=
+-Dplatforms=x11,wayland
+-Dgallium-drivers=swrast,zink
+-Dvulkan-drivers=swrast
```

Comment:
Compiling vulkan driver is useless on Termux. In Termux repository, the vulkan loader is provided by system (/system/lib(64)/libvulkan.so). IIRC it will not follow the default vulkan loader behaviors (which is, search `$PREFIX/share/vulkan/icd.d` and use them as ICD layers). It will only search some drivers in `/system/vendor` and fallback to the software implementation in Android.
If we really want to enable `swrast` as a vulkan driver, a generic vulkan loader (possibly names `vulkan-loader-generic`?) should be added too. But this deviates from the original intention of using this `zink`. The original intention of using `zink` is to bring Android native hardware acceleration to Termux, but adding such a vulkan driver will make `zink` use software rendering again.

Reply:
ok, i got